extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php

```diff
@@ -95,20 +93,6 @@ public static function onPageHistoryBeforeList() {
 		return true;
 	}
 
-	public static function onSendGridPostbackLogEvents( $events ) {
-		foreach ( $events as $event ) {
-			if ( self::isAppreciationEmailEvent( $event ) ) {
-				if ( preg_match( '/diff=([0-9]*)/', $event[ 'url' ], $diff ) ) {
-					self::sendDataToDW( 'button_clicked', $event[ 'wikia-email-city-id' ], $diff[ 1 ] );
-				} elseif ( preg_match( '/rev_id=([0-9]*)/', $event[ 'url' ], $revId ) ) {
-					self::sendDataToDW( 'diff_link_clicked', $event[ 'wikia-email-city-id' ], $revId[ 1 ] );
-				}
-			}
-		}
-
-		return true;
-	}
-
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		if ( self::shouldDisplayAppreciation() ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)